js/cypress/integration/e2e-test/fluentComponents.js

```diff
@@ -357,14 +342,6 @@ describe('Dropdown.shinyInput()', () => {
   it('value change works', () => {
     dropdownChangeTest();
   });
-
-  it('setting default values for multiSelect works', () => {
-    dropdownMultiselectDefaultTest();
-  });
-
-  it('updating multiSelect options and values works', () => {
-    dropdownMultiselectChangeTest();
-  });
 });
```

Comment:
Well, I started observing some intermittency in those tests, which I don't quite understand. It seems that now it takes a bit longer to spin up the test app and that's why some fail because components are not yet rendered. For this multiselect when I run it interactively everything works fine, but on the whole Cypress run it tends to fail -- I have big troubles debugging it.

I'd prefer to refactor those tests to be more isolated, as all components being put in one big app seems to be the root cause of this intermittency.  But for now I'm a bit stuck with this test, I'll give it one more try

Reply:
I'm confused @averissimo… Why it passed now when if didn't on multiple runs previously?

I restored multiselect tests, but since it required some changes in shiny.react, here is a [PR](https://github.com/Appsilon/shiny.react/pull/75) that enables this fix, PTAL.

Reply:
I agree with refactoring. But that's out of scope for this PR :grin: 

The main issue I found was that it was running the test sequentially and keeping the state between tests. (hence the `beforeEach(() => cy.reload())` I had on my diff